xmppserver/src/main/webapp/WEB-INF/web.xml

```diff
@@ -155,6 +160,12 @@
         <url-pattern>/server-properties.jsp</url-pattern>
     </servlet-mapping>
 
+    <servlet-mapping>
+        <servlet-name>SystemCacheDetails</servlet-name>
+        <!-- NB. .jsp extension is only required so filters are applied -->
```

Comment:
That includes the Auth filter, right? This new servlet cannot be used by non-Admins?